hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.module.scss

```diff
@@ -0,0 +1,24 @@
+@import "./../../assets/abstracts/mixins";
```

Comment:
You should use absolute imports with src as the reference point 
![image](https://github.com/ieeeuoft/hackathon-template/assets/43456930/b63f29b0-eef4-411f-b196-89989030eb61)
For this example it would be `@import "assets/abstracts/mixins"`

Reply:
Sounds good, thanks, fixed